### PR TITLE
Added code samples for virtual terminal

### PR DESCRIPTION
**Pull request by monica-m-ps**
These code samples demonstrate virtual terminal where `transactionSource` is `MOTO`.

### CODE REVIEW

**Review by luizsilva-dojo — APPROVED** [review]

File: virtual-terminal/curl/virtual-terminal.sh [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+# for secure testing.
+
+curl -v --request POST \
+    --url https://api.dojo.tech/payment-intents \
+    --header 'content-type: application/json' \
+    --header 'authorization: Basic sk_sandbox_c8oLGaI__msxsXbpBDpdtwJEz_eIhfQoKHmedqgZPCdBx59zpKZLSk8OPLT0cZolbeuYJSBvzDVVsYvtpo5RkQ' \
+    --header 'version: 2022-07-04' \
+    --data '{"amount":{"value":1000,"currencyCode":"GBP"},
+                "reference":"Order 234","transactionSource": "MOTO"
+                }'
```
Comment (RomanKondratyev) [inline virtual-terminal/curl/virtual-terminal.sh:13]: This should be aligned properly, like in the other examples (existing)

File: virtual-terminal/powershell/virtual-terminal.ps1 [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    -Headers @{ "version" = "2022-07-04"; "authorization" = "Basic $publicSandboxKey" } `
+    -ContentType 'application/json' `
+    -Body '{ "amount": 
+                { 
+                    "value": 1000, 
+                    "currencyCode": "GBP" 
+                }, 
+            "reference": "Order 234", 
+            "transactionSource": "MOTO",
+            }'
```
Comment (RomanKondratyev) [inline virtual-terminal/powershell/virtual-terminal.ps1:19]: Same here

File: virtual-terminal/python/virtual-terminal.py [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+})
+headers = {
+    'content-type': "application/json",
+    'version': "2022-04-07",
+    'authorization': "Basic sk_sandbox_c8oLGaI__msxsXbpBDpdtwJEz_eIhfQoKHmedqgZPCdBx59zpKZLSk8OPLT0cZolbeuYJSBvzDVVsYvtpo5RkQ"  # <-- Change to your secret key
+}
+conn.request("POST", "/payment-intents/", payload, headers)
+res = conn.getresponse()
+data = res.read()
+last_id = json.loads(data)["id"]
```
Comment (RomanKondratyev) [inline virtual-terminal/python/virtual-terminal.py:26]: Why do we build this `url` here?

**Comment by RomanKondratyev** [comment]
We would also need to write a section in docs about VT and how to use it via the APIs.

**Comment by monica-m-ps** [comment]
> We would also need to write a section in docs about VT and how to use it via the APIs.

Say no more... 😄 created these topics... https://github.com/dojo-engineering/remote-payments-developer-docs/pull/152

**Review by AndreiMihalciuc — APPROVED** [review]

**Pull request merged**